### ISSUE TITLE
[waiting on pyvesync] Add support for LAP-C302S-WUSB

### ISSUE THREAD

**Issue by alpharesearch** [vlebourl/custom_vesync#56]
### Home Assistant log

<details><summary>Logs</summary>

```
2022-10-31 18:17:49.969 DEBUG (SyncWorker_6) [pyvesync.helpers] [post] calling '/cloud/v1/deviceManaged/devices' api
2022-10-31 18:17:50.182 DEBUG (SyncWorker_6) [pyvesync.vesync] Unknown device named Markus Hepa Fan model LAP-C302S-WUSB
2022-10-31 18:17:50.182 DEBUG (SyncWorker_6) [pyvesync.vesync] Error - 'VeSync' object has no attribute 'unknown'
2022-10-31 18:17:50.183 DEBUG (SyncWorker_6) [pyvesync.vesync] LAP-C302S-WUSB device not added
2022-10-31 18:17:50.183 DEBUG (SyncWorker_6) [pyvesync.helpers] [post] calling '/cloud/v2/deviceManaged/bypassV2' api
2022-10-31 18:17:50.472 DEBUG (SyncWorker_6) [pyvesync.helpers] [post] calling '/cloud/v2/deviceManaged/bypassV2' api
```

</details>

### Additional information

Looks like to me this device is missing at the pyvesync library that is used with this integration. That's why I opened this issue as well: https://github.com/webdjoe/pyvesync/issues/152

But I may be wrong and I was wondering if there is any way to overwrite this version with the standard Core 300S version?

**Comment by alpharesearch** [comment]
I just updated everything and I don't see any difference. Is there any way to force this as a core 300s device?

**Comment by vlebourl** [comment]
Hi, sorry I was on holidays. You are correct, your device is not in the underlying library... nothing I can do on my side for this. Let's wait for @webdjoe's answer on your issue. The device needs to be added here https://github.com/webdjoe/pyvesync/blob/8861ec5fe27db9b3734744f6c31602fb10e42c55/src/pyvesync/vesyncfan.py#L68, maybe you can open a pull request to make things faster? 
I'll close this one for now. Don't hesitate to reopen once added to the library if it doesn't work here.
Cheers

**Comment by Drewlrox** [comment]
Hello, I am new to how github works. But it appears this may have been added here. https://github.com/webdjoe/pyvesync/issues/152

I am closing my duplicate issue I had posted before seeing this.

**Comment by vlebourl** [comment]
waiting on https://github.com/webdjoe/pyvesync/pull/160

**Comment by alpharesearch** [comment]
I have the latest HA and I don't see any update on HACS either. I guess the new library needs to be published before I can test it...

**Comment by vlebourl** [comment]
yes indeed...

**Comment by Drewlrox** [comment]
It seems pyvesync had a release 4 days ago, webdjoe had stated support would be in the next release 6 weeks ago. Maybe it was added now? I am unsure how to check.

**Comment by KyleStilkey** [comment]
> It seems pyvesync had a release 4 days ago, webdjoe had stated support would be in the next release 6 weeks ago. Maybe it was added now? I am unsure how to check.

I believe the last update has added these support for more devices including the air fryers. I also do know they are trying to get a lot of this to just be integrated with the official integration but that's been another issue. Hopefully we see this updated and maybe the core version will finally get updated with all the changes made here and pyvesync. It's just a waiting game at this point right now for support to be added.

**Comment by vlebourl** [comment]
The custom integration will be updated soon to account for latest pyvesync version. 
Let me know if that fixes your issues

**Comment by alpharesearch** [comment]
I installed the latest stable version and the hepa filter fan is now found. Thanks.  

As expected all the standard functions work, but this unit has one more feature (plasma pro) that can be turned on or off. I guess next step  would be to expose this switch. What information can I provide to assist with that?

**Comment by vlebourl** [comment]
Sorry I didn't see this message... 
Can you open a new feature request for the plasma pro function? 
Provide the diagnostics file with it and I'll try to add it. If not possible with only the diagnostics file, I might request access to your device if you're ok with that, but we'll discuss that later. 
Cheers.
